src/test/e2e/tests/seal.spec.ts

```diff
@@ -46,13 +48,25 @@ test('Lock MKR, select rewards, select delegate, and open position', async ({ pa
   await expect(page.getByText('Confirm your position').nth(0)).toBeVisible();
   await expect(page.getByTestId('widget-container').getByText('Sealing')).toBeVisible();
   await expect(page.getByText('100 MKR')).toBeVisible();
+  await expect(page.getByTestId('widget-container').getByText('Borrowing')).toBeVisible();
+  await expect(page.getByText('38,000 USDS')).toBeVisible();
   await expect(page.getByTestId('widget-container').getByText('Seal reward')).toBeVisible();
 
   await approveOrPerformAction(page, 'Approve seal amount');
   expect(page.getByRole('heading', { name: 'Token access approved' })).toBeVisible();
   await approveOrPerformAction(page, 'Continue');
   expect(page.getByRole('heading', { name: 'Success!' })).toBeVisible();
-  await expect(page.getByText("You've sealed 100 MKR. Your new position is open.")).toBeVisible();
+  // await expect(page.getByText("You've sealed 100 MKR. Your new position is open.")).toBeVisible();
   await page.getByRole('button', { name: 'Manage your position(s)' }).click();
   await expect(page.getByText('Position 1')).toBeVisible();
+
+  await page.getByRole('button', { name: 'Manage Position' }).click();
+
+  await expect(page.getByText('Your position 1')).toBeVisible();
+  // await expect(page.getByTestId('borrow-input-lse-balance')).toHaveText('Limit 0 <> 17,593 USDS');
+
+  await page.getByTestId('borrow-input-lse').fill('100');
+
+  // TODO: Insufficient balance error appears, but it shouldn't
+  await expect(page.getByText('Insufficient collateral')).not.toBeVisible();
```

Comment:
This part of the test would pass when this is fixed https://github.com/jetstreamgg/helipad/pull/890